core/rpc/src/state/tests.rs

```diff
@@ -28,11 +28,45 @@ fn should_return_storage() {
 	let client = Arc::new(test_client::new());
 	let genesis_hash = client.genesis_hash();
 	let client = State::new(client, Subscriptions::new(core.executor()));
+	let key = StorageKey(b":code".to_vec());
 
+	assert!(
+		client.storage(key.clone(), Some(genesis_hash).into())
+			.map(|x| x.map(|x| x.0.len())).unwrap().unwrap()
+		> 195_000
```

Comment:
What does this check check?

Reply:
We check that the runtime code is there and the size roughly matches - I don't think we want to depend on a specific runtime code or specific size (that would break the test on every change).

Do you suggest checking a different entry or using a different condition here?

Reply:
No-no, I was curious what is the number. But mb a comment wouldn’t harm?